app/models/tutorial.rb

```diff
@@ -63,22 +99,33 @@ def get_category_notice
     return notice
   end
 
+  # Gets draft title for a tutorial
+  #
+  # If the tutorial is a draft:
+  # => Returns a String: "DRAFT: tutorial.title.upcase"
+  # If the tutorial is not a draft:
+  # => Returns a String "EDIT: tutorial.title.upcase"
   def get_draft_title_for_tutorial
     return self.draft == true ? "DRAFT: #{self.title.upcase}" : "EDIT: #{self.title.upcase}"
   end
 
-def create_chapters(params)
+  # Creates new chapters.
+  #
+  # Takes params arguments.
+  # 
+  # Returns a new Chapter Object.
+  def create_chapters(params)
     #@new_chapters = []
     #params.each do |key, value|
-      @new_chapter = self.chapters.new(title: params[:title], description: params[:description], draft: params[:draft])
+    @new_chapter = self.chapters.new(title: params[:title], description: params[:description], draft: params[:draft])
       #@new_chapters << chapter 
-    end
+  end
 
-def save_if_valid
-  if @new_chapter.valid?
-    @new_chapter.save
+  def save_if_valid
+    if @new_chapter.valid?
```

Comment:
Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.